workspace/__lib__/xod-dev/esp8266-mcu/esp8266-mcu-inet/patch.cpp

```diff
@@ -0,0 +1,17 @@
+
+// clang-format off
+{{#global}}
+#include <ESP8266WiFi.h>
+{{/global}}
+// clang-format on
+
+struct State {
+};
+
+using Type = ESP8266WiFiClass*;
+
+{{ GENERATED_CODE }}
+
+void evaluate(Context ctx) {
```

Comment:
`warning: unused parameter 'ctx'`

Reply:
Removed `evaluate` entirely, so if someone will use this node directly the error will be produced at compile time. @nkrkv, do you think it is ok?